### ISSUE TITLE
Formatting of log on Windows

### ISSUE THREAD

**Issue by kambrium** [andrewlalis/handy-httpd#13]
The log looks fine on Linux. However, on Windows it's formatted weirdly. See the following excerpt from Power Shell with all those blanks.

```
←   [   3   3   m   h   a   n   d   y   _   h   t   t   p   d   .   s   e   r   v   e   r   ←   [   0   m                                ←   [   3   4   ;   1   m   I   N   F   O   ←   [   0   m    ←[90m2023-03-19T12:30:50.246←[0m Bound to address 127.0.0.1:4343
←   [   3   3   m   h   a   n   d   y   _   h   t   t   p   d   .   s   e   r   v   e   r   ←   [   0   m                                ←   [   3   4   ;   1   m   I   N   F   O   ←   [   0   m    ←[90m2023-03-19T12:30:50.246←[0m Now accepting connections.
←   [   3   3   m   .   .   .   t   p   d   .   c   o   m   p   o   n   e   n   t   s   .   w   o   r   k   e   r   ←   [   0   m        ←   [   3   4   ;   1   m   I   N   F   O   ←   [   0   m    ←[90m2023-03-19T12:31:38.538←[0m GET
←   [   3   3   m   .   .   .   t   p   d   .   c   o   m   p   o   n   e   n   t   s   .   w   o   r   k   e   r   ←   [   0   m        ←   [   3   4   ;   1   m   I   N   F   O   ←   [   0   m    ←[90m2023-03-19T12:31:38.541←[0m 200 OK
```

What could be the reason?

**Comment by andrewlalis** [comment]
Looks like it could be an issue with how the default logging provider formats colored messages (since I'm just using ANSI color codes which may not be 100% platform independent yet). You can remove the color formatting though by manually configuring the logging provider your application uses, instead of letting it decide automatically: https://andrewlalis.github.io/slf4d/guide/configuring.html

Basically, add this to the start of your `main` function:
```d
import slf4d;
import slf4d.default_provider;
auto provider = new shared DefaultProvider(false, Levels.TRACE);
configureLoggingProvider(provider);
```

**Comment by kambrium** [comment]
Thank you. This partially solved it. Some blanks are left though.

```
h   a   n   d   y   _   h   t   t   p   d   .   s   e   r   v   e   r                                I   N   F   O    2023-03-19T20:28:37.604 Bound to address 127.0.0.1:4343
h   a   n   d   y   _   h   t   t   p   d   .   s   e   r   v   e   r                                I   N   F   O    2023-03-19T20:28:37.604 Now accepting connections.
```

**Comment by andrewlalis** [comment]
Looks like this is an issue with Phobos' `padLeft` and `padRight` functions. I'll investigate further tomorrow.

**Comment by andrewlalis** [comment]
In the meantime, instead of SLF4D's default provider, I can also recommend [slf4d-dlog](https://code.dlang.org/packages/slf4d-dlog), which forwards log messages to Tristan Kildare's [DLog](https://code.dlang.org/packages/dlog) logger. I'll need to spin up a windows instance this evening to actually be able to test this behavior.

**Comment by andrewlalis** [comment]
okay, I am not able to reproduce the issue on my windows installation. I'll make some changes to my SLF4D library to disable colored output by default to hopefully improve compatibility. If you still do encounter the error, please open an issue in the [SLF4D](https://github.com/andrewlalis/slf4d) repository with more details about your OS/terminal/etc.

**Comment by kambrium** [comment]
Ok, I'll check on the weekend.
